interface/modules/custom_modules/oe-module-faxsms/src/Controller/AppDispatch.php

```diff
@@ -0,0 +1,537 @@
+<?php
```

Comment:
placeholder for ongoing review (i've passed @adunsulag placeholder, woohoooo!). unless unforeseen events happen, should complete review late tonight :)

Reply:
I didn't realize how large this module was but thanks for taking time for diligent review.
By time you come back to this I'll have the last two features in. Namely fax forward(email and new fax number) and chart doc.